Objective-C/ChartboostExample/File.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  File.swift
+//  ChartboostExample
+//
+//  Created by Javier Serrano Muñoz on 18/5/22.
+//  Copyright © 2022 chartboost. All rights reserved.
+//
+
+import Foundation
```

Comment:
a solution for this is to include the swift libs in the search paths:
```
$(SDKROOT)/usr/lib/swift
$(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)
```

Reply:
I already tried that. Still fails.

Reply:
I think it's cleaner than having an empty swift file in a objc project

Reply:
I can see your point. What do you think about this: Instead of leaving it empty just have a comment in file explaining why does the file exist.

Reply:
I see it worked for me in the sim but not in a physical device, happened the same? I guess if we can not make it work that way, we can add a comment in this file explaining it.

Reply:
For me adding this: 

$(SDKROOT)/usr/lib/swift
$(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)

It Builds, but it fails to execute in simulator. I have not tried in real device. I tried several things, the only thing that works to me is adding the swift file. I even found this very old article https://developer.apple.com/library/archive/qa/qa1881/_index.html which led to to try to set "Always embed swift standard libraries" which for me has a lot sense but still does not work.

Reply:
found a solution!  
it was missing adding the **/usr/lib/swift** to "Runpath Search Path"
to solve the problem on runtime **Library not loaded: @rpath/libswiftCore.dylib**